meta/src/validator.rs

```diff
@@ -115,27 +114,6 @@ pub fn validate_pairs(pairs: Pairs<'_, Rule>) -> Result<Vec<&str>, Vec<Error<Rul
     Ok(defaults.cloned().collect())
 }
 
-/// Validates that the given `definitions` do not contain any Rust keywords.
-#[allow(clippy::ptr_arg)]
-pub fn validate_rust_keywords(definitions: &Vec<Span<'_>>) -> Vec<Error<Rule>> {
```

Comment:
Oof didn't catch that, I glossed over the `pub` since I thought this would be internal just from how obscure it is. I'll bring it back and mark it as deprecated